### PR TITLE
modal updated

### PR DESCRIPTION
**Pull request by kschmott**

### CODE REVIEW

**Comment by santiagomed** [comment]
Looks good so far! One thing: dark-mode colors make it tough on the visibility: 
image
Maybe switch the font color to white on dark mode?

**Comment by kschmott** [comment]
> Looks good so far! One thing: dark-mode colors make it tough on the visibility: image Maybe switch the font color to white on dark mode?

I did not even know we were doing dark mode. Did the designers add a darkmode modal?

File: src/components/shared/SignupModal.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <div className="modal-box relative">
+          <label
+            htmlFor="my-modal-3"
+            onClick={() => onClose()}
+            className="btn btn-sm btn-circle absolute right-2 top-2 text-black bg-white border-none hover:bg-gray-400"
+          >
+            ✕
+          </label>
+          <h3 className="text-4xl font-bold mx-auto w-fit text-primary">
+            Join <a className="text-secondary">M</a>Clubs
```
Comment (santiagomed) [inline src/components/shared/SignupModal.tsx]: I'm assuming the "M" is going to be a link in the future? If it is then it looks like eslint is giving errors here. You can look over the check's output, but if I'm not mistaken you are missing an `href` attribute here. Something like `href="/"` as a temporary measure should be fine.

If you are not planing the "M" to be a link, then I'd recommend switching it to a span or something.

**Comment by santiagomed** [comment]
> I did not even know we were doing dark mode. Did the designers add a darkmode modal?

It looks like if the user has dark mode turned on in their machine, it turns into dark mode. It's probably a tailwind feature. I don't think it's a bad idea so we can ask the designers to come up with some quick designs for it tomorrow.

File: src/components/shared/SignupModal.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              <li className="my-1">Write reviews</li>
+              <li className="my-1">Submit questions to clubs</li>
+            </ul>
+          </div>
+          <div className="flex flex-col justify-center items-center gap-4 ">
+            <button className="btn btn-outline max-w-xs w-full text-primary hover:bg-primary font-inter text-base">
+              Login with UMID
+            </button>
+            <button className="btn bg-primary text-white max-w-xs w-full hover:bg-white hover:text-primary font-inter text-base">
+              Create Account with UMID
```
Comment (santiagomed) [inline src/components/shared/SignupModal.tsx]:
```suggestion
              Create an Account with your UM email
```
I'd reword this. If we say "UMID" the user might think it's through the UM weblogin, which isn't the case. I'm guessing you guys followed the designs, so we gotta sync with the designers tomorrow.

File: src/components/shared/SignupModal.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div>
+      <label
+        htmlFor="my-modal-3"
+        className={`modal cursor-pointer ${isOpen && "modal-open"}`}
+      >
+        <div className="modal-box relative">
+          <label
+            htmlFor="my-modal-3"
+            onClick={() => onClose()}
```
Comment (santiagomed) [inline src/components/shared/SignupModal.tsx]:
```suggestion
            onClick={onClose}
```
nit: you can just pass it as a callback

File: src/App.tsx [hunk 4]
```diff
@@ -1,26 +1,22 @@
...
-          </p>
-          <div className="modal-action">
-            <label htmlFor="my-modal" className="btn">
-              Yay!
-            </label>
-          </div>
-        </div>
-      </div>
+
+      <SignupModal isOpen={modal} onClose={() => setModal((prev) => !prev)} />
```
Comment (santiagomed) [inline src/App.tsx]:
```suggestion
      <SignupModal isOpen={modal} onClose={() => setModal(false)} />
```
nit

**Review by SrishVU — APPROVED** [review]

**Review by meganparada9 — APPROVED** [review]

**Review by santiagomed — APPROVED** [review]
